src/NhekoDBusInterface.h

```diff
@@ -0,0 +1,65 @@
+// SPDX-FileCopyrightText: 2022 Nheko Contributors
```

Comment:
Can you move the 4 dbus files into a dbus subfolder?

Reply:
Sure